test/libyul/objectCompiler/long_object_name.yul

```diff
@@ -10,16 +10,18 @@ object "t" {
 // optimizationPreset: full
 // ----
 // Assembly:
-//     /* "source":23:147   */
+//     /* "source":33:146   */
 //   dataSize(sub_0)
 //     /* "source":30:31   */
 //   0x00
 //     /* "source":23:147   */
 //   sstore
+//   stop
 // stop
 //
 // sub_0: assembly {
+//       stop
 // }
-// Bytecode: 6000600055fe
-// Opcodes: PUSH1 0x0 PUSH1 0x0 SSTORE INVALID
-// SourceMappings: 23:124:0:-:0;30:1;23:124
+// Bytecode: 600160005500fe
+// Opcodes: PUSH1 0x1 PUSH1 0x0 SSTORE STOP INVALID
+// SourceMappings: 33:113:0:-:0;30:1;23:124;
```

Comment:
Note to self: this is something to watch out for - apparently we assume that the code walks past the end of the bytecode and rely on that for generating ``INVALID``... or not sure what else is happening here...

Reply:
For the record: this is ``libevmasm/Assembly.cpp`` line 756.
Interesting, actually, not sure what we should do here or whether we should do anything.
The constructor code is always expected to return something, so walking past the outermost block of constructor code shouldn't happen.
I'm not sure if we have formally defined how Yul code behaves when walking past the outermost block - that would affect this. Generating ``STOP`` there seems reasonable to me, but it breaks this safeguard...